src/Bridge/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -61,6 +61,7 @@
             <argument type="service" id="api_platform.identifiers_extractor.cached" />
             <argument type="service" id="api_platform.subresource_data_provider" on-invalid="ignore" />
             <argument type="service" id="api_platform.identifier.converter" on-invalid="ignore" />
+            <argument type="service" id="request_stack" />
```

Comment:
I don't think we want to have a dependency on the `RequestStack` here. We're aiming for support of other frameworks.